cdap-watchdog-api/src/main/java/co/cask/cdap/api/metrics/MetricStore.java

```diff
@@ -53,10 +53,19 @@
 
   /**
    * Deletes all metric data before given timestamp. Used for applying TTL policy.
+   *
    * @param timestamp time up to which to delete metrics data, in ms since epoch
    */
   void deleteBefore(long timestamp) throws Exception;
```

Comment:
What does this method do now?

Reply:
This method deletes all the resolution tables except the total resolution before the given timestamp, it is now only used to clean up all the metrics from unit test.